src/util/configHelpers.js

```diff
@@ -1049,7 +1077,7 @@ const mergeListingConfig = (hostedConfig, defaultConfigs, categoriesInUse) => {
 
   // When debugging, include default configs by passing 'true' here.
   // Otherwise, use listing types and fields from hosted assets.
-  const shouldMerge = mergeDefaultTypesAndFieldsForDebugging(false);
+  const shouldMerge = mergeDefaultTypesAndFieldsForDebugging(true);
```

Comment:
Don't commit this one

Reply:
Hmm, should I amend the original commit?

Reply:
Yes. Create a new fixup commit and merge it using interactive rebasing. (I can help)